app/modules/training/http_training.html

```diff
@@ -0,0 +1,14 @@
+<html>
+  <body>
+    <h1>HTTP Training</h1>
+    {% if title %}
+    <p>Latitude: {{ lat }}</p>
```

Comment:
`lat` and `lon` do not contain any values. Please either include values for these template variables or remove this section from the template.